cclib/parser/orcaparser.py

```diff
@@ -771,9 +771,12 @@ def splitter(line):
                 self.etsecs.append(sec)
                 line = next(inputfile)
 
-
         # Parse the various absorption spectra for TDDFT and ROCIS
         if 'ABSORPTION SPECTRUM' in line or 'ELECTRIC DIPOLE' in line:
+            # CASSCF has an anomalous printing of ABSORPTION SPECTRUM
+            if line[:-1] == 'ABSORPTION SPECTRUM':
+                return
```

Comment:
Why isn't this `break`?

Reply:
It isn't inside a loop. The function extract is called inside a loop over the lines of the inputfile (line 311 of logpfileparser.py)

Reply:
Eric's point if that break allows the parser to potentially trigger off the same line... so I also think `break` is better.